targets/ESP32/ESP32_C61/target_common.c

```
@@ -0,0 +1,29 @@
+
+    1, // ConvertCOM_DebugHandle(1),
+    0, // ConvertCOM_DebugHandle(0),
+    921600,
+    0, // STDIO = COM2 or COM1
+
+    {RAM1_MEMORY_StartAddress, RAM1_MEMORY_Size},
+    {FLASH1_MEMORY_StartAddress, FLASH1_MEMORY_Size}};
+
+HAL_TARGET_CONFIGURATION g_TargetConfiguration;
+
+void FixUpHalSystemConfig()
```

Cppcheck (2.20.0)

[style] The function 'FixUpHalSystemConfig' is never used.

(unusedFunction)